added info about with_current_organizations to the person docs (#115)
affinity/slate: source/includes/_persons.md

| primary_email            | string      | The email (automatically computed) that is most likely to the current active email address of the person.                                                                                                                                                                                            |
| organization_ids         | integer[]   | An array of unique identifiers of organizations that the person is associated with.                                                                                                                                                                                                                  |
| opportunity_ids          | integer[]   | An array of unique identifiers of opportunities that the person is associated with.                                                                                                                                                                                                                  |

> | current_organization_ids | integer[]   | An array of unique identifiers of companies that the person is associated with and are indicated as their current companies.                                                                                                                                                                         |

Reviewer: ```suggestion
| current_organization_ids | integer[]   | An array of unique identifiers of organizations that the person is currently associated with according to the Affinity Data: Current Organization in-app column.                                                                                                                                                                       |
```

| term                            | string  | false    | A string used to search all the persons in your team's address book. This could be an email address, a first name or a last name.                                                                                                                   |
| with_interaction_dates          | boolean | false    | When true, interaction dates will be present on the returned resources. Only persons that have interactions will be returned.                                                                                                                       |
| with_interaction_persons        | boolean | false    | When true, persons for each interaction will be returned. Used in conjunction with `with_interaction_dates`                                                                                                                                         |

> | with_opportunities              | boolean | false    | When true, opportunity IDs associated with this person will be returned.                                                                                                                                                                            |

Reviewer: ```suggestion
| with_opportunities              | boolean | false    | When true, opportunity IDs will be returned for each person.                                                                                                                                                                            |
```

> | with_current_organizations      | boolean | false    | When true, the IDs of person's current organizations will be returned.                                                                                                                                                                              |

Reviewer: ```suggestion
| with_current_organizations      | boolean | false    | When true, the organization IDs of each person's current organizations (according to the Affinity Data: Current Organizations column) will be returned.                                                                                                                                                                              |
```

### Path Parameters

| Parameter                  | Type    | Required | Description                                                                                                 |
| -------------------------- | ------- | -------- | ----------------------------------------------------------------------------------------------------------- |
| person_id                  | integer | true     | The unique ID of the person that needs to be retrieved.                                                     |
| with_interaction_dates     | boolean | false    | When true, interaction dates will be present on the returned resources.                                     |
| with_interaction_persons   | boolean | false    | When true, persons for each interaction will be returned. Used in conjunction with `with_interaction_dates` |
| with_opportunities         | boolean | false    | When true, opportunity IDs associated with this person will be returned.                                    |

> | with_current_organizations | boolean | false    | When true, the IDs of person's current organizations will be returned.                                      |

Reviewer: ```suggestion
| with_current_organizations | boolean | false    | When true, the IDs of this person's current organizations (according to the Affinity Data: Current Organizations column) will be returned.                                      |
```
